### PR TITLE
Bump to 1.8.0

### PR DESCRIPTION
**Pull request by kylekthompson**
Prepare 1.8.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -119,26 +125,26 @@ At this time, loading test runs from other queue instances includes the
...
-    remote test run state will not be loaded. Executing a test suite whose
-    run state file failed to be loaded will fall back on executing the test
-    suite as a fresh run, similar to the pre-1.4.0 behavior.
+  guarantees across versions of ABQ queues are provided at this time.
+  `run_state` files are guaranteed to be compatible if shared between ABQ
+  queues of the same version.
+  If an ABQ queue loads a `run_state` file that it is incompatible with, the
+  remote test run state will not be loaded. Executing a test suite whose
+  run state file failed to be loaded will fall back on executing the test
+  suite as a fresh run, similar to the pre-1.4.0 behavior.
```
Comment (kylekthompson) [inline CHANGELOG.md:134]: Prettier decided to fix indentation of this

File: CHANGELOG.md [hunk 2]
```diff
@@ -199,11 +207,12 @@ remote storage on `abq start`:
...
-        - `<mode>` is either "store" or "load", depending on whether the file should be stored
+
+      - `<mode>` is either "store" or "load", depending on whether the file should be stored
         into the remote location, or loaded from the remote location.
-        - `<file-type>` is either "manifest" or "results".
-        - `<run-id>` is the run ID of the test suite run.
-        - `<local-path>` is the path to the file on the local filesystem. If the mode is "store",
+      - `<file-type>` is either "manifest" or "results".
+      - `<run-id>` is the run ID of the test suite run.
+      - `<local-path>` is the path to the file on the local filesystem. If the mode is "store",
```
Comment (kylekthompson) [inline CHANGELOG.md:215]: Again, prettier. Not sure if this one is correct or incorrect though \cc @ayazhafiz

Comment (ayazhafiz) [reply CHANGELOG.md:215]: I looked at the render and this seems good. I don't think we have prettier configured on this repo though maybe we should.

**Review by ayazhafiz — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (kylekthompson) [reply CHANGELOG.md:215]: Yea, sorry it's in my editor 😅 I didn't even realize it made these changes until I looked at the diff

**Pull request merged**